Gemfile

```diff
@@ -54,12 +54,14 @@ group :system_tests do
   gem 'serverspec',                         :require => false
   gem 'beaker-hostgenerator', '>= 1.1.22',  :require => false
   gem 'beaker-docker',                      :require => false
+  gem 'beaker-vagrant',                     :require => false
   gem 'beaker-puppet',                      :require => false
   gem 'beaker-puppet_install_helper',       :require => false
   gem 'beaker-module_install_helper',       :require => false
   gem 'rbnacl', '>= 4',                     :require => false if RUBY_VERSION >= '2.2.6'
   gem 'rbnacl-libsodium',                   :require => false if RUBY_VERSION >= '2.2.6'
   gem 'bcrypt_pbkdf',                       :require => false
+  gem 'beaker-vagrant',                     :require => false
```

Comment:
can you remove one `beaker-vagrant` entry?

Reply:
Rebase gone wrong. Initially I did manually and then the proper way so modulesync wouldn't be unhappy with me.